Sources/Extensions/ColletionExtension.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ColletionExtension.swift
+//  HandySwift iOS
+//
+//  Created by Stepanov Pavel on 08/07/2018.
+//  Copyright © 2018 Flinesoft. All rights reserved.
+//
+
+import Foundation
+
+extension Collection {
+    
+    /// Returns an element with the specified index or nil if the element does not exist
```

Comment:
Please set a `.` at the end of this line since this is a valid english sentence.